homework_12/src/js/elementBuilder.js

```diff
@@ -0,0 +1,8 @@
+import $ from 'jquery';
+
+export default (tag = 'div', content = '', className = 'box') => {
+  
+    const element = $(`<${tag}>`).addClass(className).html(content)
```

Comment:
why you have additional indentation before const?
also semicolon should be at the end of expression